### PR TITLE
PLT-7189: fix missing message issue and upgrade to new logger client

### PR DESCRIPTION
**Pull request by chainat-byb**
Ticket: https://beforeyoubid.atlassian.net/browse/PLT-7189

Please see the ticket description/acceptance criterion for changes I am introducing.

Thanks

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/app/
   docker:
-    - image: cimg/node:14.19
+    - image: cimg/node:16.20.0
```
Comment (Dmitrii8723-byb) [inline .circleci/config.yml:4]: Tiny: potentially moving to node 18

**Review by Dmitrii8723-byb — APPROVED** [review]
L1/L2 review. Looking great @chainat-byb

File: .circleci/config.yml (hunk 1 above)
Comment (chainat-byb) [reply .circleci/config.yml:4]: Thanks Dima, let's do them all together as part of node 18 migration or node 20 if we can

**Review by mitch-byb — APPROVED** [review]
Awesome documentation too 👍

**Pull request merged**